src/state/README.md

```diff
@@ -52,3 +52,103 @@ app.container.on('connected', async () => {
     await UserData.delete('my-value');
 });
 ```
+### IndexedDB
```

Comment:
We should probably include an external link out to the MDN page for IndexedDB (or whatever page[s] you find is the most informative) just to include some info about the system outside of Springroll